### PR TITLE
[MIG] auth_signup_verify_email: Migration to 13.0

### PR DESCRIPTION
**Pull request by em230418**

### CODE REVIEW

File: auth_signup_verify_email/README.rst [hunk 1]
```diff
@@ -86,6 +86,7 @@ Contributors
 * Jairo Llopis <yajo.sk8@gmail.com>
 * Simone Orsi <simone.orsi@camptocamp.com>
 * Alexandre Díaz <alexandre.diaz@tecnativa.com>
+* Eugene Molotov <molotov@it-projects.info>
```
Comment (yelizariev) [inline auth_signup_verify_email/README.rst]:
```
.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
   !! This file is generated by oca-gen-addon-readme !!
   !! changes will be overwritten.                   !!
   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

Comment (em230418) [reply auth_signup_verify_email/README.rst]: Removed changes on this file

Comment (yelizariev) [reply auth_signup_verify_email/README.rst]: You can update readme/CONTRIBUTORS.rst

**Comment by CarlosRoca13** [comment]
Please, separate pre-commit changes from the migration ones to ease furtther review. And please, run pre-commit. 

You can get help on this link https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0

**Comment by em230418** [comment]
> Please, separate pre-commit changes from the migration ones to ease furtther review. And please, run pre-commit.
> 
> You can get help on this link https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0

Thanks for the link above!

Btw, in instructions I ignored this
```
git format-patch --keep-subject --stdout origin/13.0..origin/12.0 -- $MODULE | git am -3 --keep
```
'cos it would produce previous commits with my signature. So I cherry-picked instead.

File: auth_signup_verify_email/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1,4 @@
+* Rafael Blasco <rafaelbn@antiun.com>
+* Jairo Llopis <yajo.sk8@gmail.com>
+* Simone Orsi <simone.orsi@camptocamp.com>
+* Alexandre Díaz <alexandre.diaz@tecnativa.com>
```
Comment (CarlosRoca13) [inline auth_signup_verify_email/readme/CONTRIBUTORS.rst:4]:
```suggestion
* Alexandre Díaz <alexandre.diaz@tecnativa.com>
* Your name and lastname <email if u want>
```
I think you would like to be a contributor

**Review by CarlosRoca13 — APPROVED** [review]
Test it on runbot and works well! 
LGTM! 
just a suggestion

File: auth_signup_verify_email/readme/CONTRIBUTORS.rst (hunk 2 above)
Comment (em230418) [reply auth_signup_verify_email/readme/CONTRIBUTORS.rst:4]: Replaced https://github.com/OCA/server-auth/pull/198/commits/562c331bd7c22551ae06ab1a1b93584a09aabf36 with https://github.com/OCA/server-auth/pull/198/commits/2aac248a524cde20804028f3022e936d95eb79c8

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**